lib/bios/menus.robot

```diff
@@ -535,13 +536,36 @@ Remove Disk Password
     END
     Press Key N Times    1    ${SETUP_MENU_KEY}
 
+Send Reboot Command
+    Telnet.Write Bare    \x1bR\x1br\x1bR
+
 Tianocore Reset System
-    # EDK2 interprets Alt + Ctrl + Del on USB keyboards as reset combination.
-    # On serial console it is ESC R ESC r ESC R.
     IF    '${DUT_CONNECTION_METHOD}' == 'SSH'
         FAIL    SSH not supported for interfacing with TianoCore
     ELSE IF    '${DUT_CONNECTION_METHOD}' == 'Telnet'
-        Telnet.Write Bare    \x1bR\x1br\x1bR
+        FOR    ${i}    IN RANGE    0    5
+            ${out}=    Read From Terminal
+
+            # Clean up the string
```

Comment:
Do we need to implement it once again?

In the history of this function, we can see the "manual" method has been replaced with the CTRL+ALT+DELETE here: ee72aadba96d247500fb31c22d878c0113255e2c

If we can prove that:
- the "new" method was less reliable
- the "old" method was more reliable

We can simply bring back the old method.

On top of this, we can add some improvements, if necessary, but I would start with simply bringing the old way.

Reply:
The new method breaks from time to time: 
[Uploading log.zip…]()
![image](https://github.com/user-attachments/assets/afc01099-93c1-4d21-bf56-dba3f3b97796)

Ill restore the "old" method and check if it will resolve the issue.

Reply:
[log2.zip](https://github.com/user-attachments/files/17631741/log2.zip)
Issue still persists... I'll restore previous version and see if it will make any difference